x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/collapsed_item_actions.tsx

```diff
@@ -179,6 +182,42 @@ export const CollapsedItemActions: React.FunctionComponent<ComponentOpts> = ({
     ];
   }, [isDisabled, item, snoozedButtonText]);
 
+  const onDisableModalOpen = useCallback(() => {
+    setIsUntrackAlertsModalOpen(true);
+  }, []);
+
+  const onModalClose = useCallback(() => {
```

Comment:
Easier to understand and match with `onDisableModalOpen`
```suggestion
  const onDisableModalClose = useCallback(() => {
```